src/plugins/yamlcpp/README.md

```diff
@@ -0,0 +1,310 @@
+- infos = Information about the yamlcpp plugin is in keys below
+- infos/author = René Schwaiger <sanssecours@me.com>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage/yaml
+- infos/recommends =
+- infos/placements = getstorage setstorage
+- infos/status = maintained preview experimental unfinished concept discouraged
+- infos/metadata =
+- infos/description = This storage plugin reads and writes data in the YAML format
+
+## Introduction
+
+The YAML CPP plugin reads and writes configuration data via the [yaml-cpp][] library.
+
+## Usage
+
+You can mount this plugin via `kdb mount`:
+
+```sh
+kdb mount config.yaml /examples/yamlcpp yamlcpp
+```
+
+. To unmount the plugin use  `kdb umount`:
+
+```sh
+kdb umount /examples/yamlcpp
+```
+
+. The following examples show how you can store and retrieve data via `yamlcpp`.
+
+```sh
+# Mount yamlcpp plugin to cascading namespace `/examples/yamlcpp`
+kdb mount config.yaml /examples/yamlcpp yamlcpp
+
+# Manually add a mapping to the database
+echo "🔑 : 🐳"               > `kdb file /examples/yamlcpp`
+# Retrieve the value of the manually added key
+kdb get /examples/yamlcpp/🔑
+#> 🐳
+
+# Save the location of the config file so we can use it later
+echo `kdb file /examples/yamlcpp` > /tmp/data_file
```

Comment:
The command `kdb file` does not work properly if the syntax of a file is incorrect. I used this hack since the Markdown Shell Recorder does not support shell variables.

Reply:
Okay, we should at least store everything within the same temporary folder to avoid problems such as #1132 #287

Reply:
After commit e9f56e98 the Markdown Shell Recorder test of the YAML CPP plugins stores temporary data inside the directory `/tmp/elektra`. The test now also removes the temporary data.